packages/prerender/src/babelPlugins/babel-plugin-redwood-prerender-media-imports.ts

```diff
@@ -63,30 +58,21 @@ export default function (
 
         if (ext && options.extensions.includes(ext)) {
           const importConstName = getVariableName(p)
-          let copiedAssetPath
 
-          if (bundler !== BundlerEnum.WEBPACK) {
-            if (state.filename === undefined) {
-              return
-            }
-            const absPath = join(dirname(state.filename), p.node.source.value)
-            const viteManifestKey = ensurePosixPath(
-              relative(getPaths().web.src, absPath),
-            )
-
-            // Note: The entry will not exist if vite has inlined a small asset
-            copiedAssetPath = (buildManifest as ViteManifest)[viteManifestKey]
-              ?.file
-          } else if (bundler === BundlerEnum.WEBPACK) {
-            const webpackManifestKey = `static/media/${basename(
-              p.node.source.value,
-            )}`
-            copiedAssetPath = buildManifest[webpackManifestKey]
-          } else {
-            // We really shouldn't get here, but just in case
-            throw new Error(`Unknown bundler used: ${bundler}`)
+          if (state.filename === undefined) {
+            return
           }
+          const absPath = join(dirname(state.filename), p.node.source.value)
+          const viteManifestKey = ensurePosixPath(
+            relative(getPaths().web.src, absPath),
+          )
+
+          // Note: The entry will not exist if vite has inlined a small asset
+          const copiedAssetPath = (buildManifest as ViteManifest)[
+            viteManifestKey
+          ]?.file
 
+          // TODO(jgmw): Ask for clarification on the following comment
           // If webpack has copied it over, use the path from the asset manifest
```

Comment:
```suggestion
          // If bundler has copied it over, use the path from the asset manifest
```
This is the only change we need here really